# **Statistics Basic-2**

# **Assignment**

**Q1. What are the three measures of central tendency?**

**SOLUTION:**
the three measure of central tendency are
1. mean
2. median
3. mode

**Q2. What is the difference between the mean, median, and mode? How are they used to measure the central tendency of a dataset?**

**SOLUTION:**

Mean: Measures the arithmetic average and is best when you need an overall "balance" of the data. It considers every value but can be skewed by extreme outliers.

Median: Measures the middle value in a dataset. It gives a better central point when there are outliers or when the data is skewed.

Mode: Measures the most common or frequent value. It is particularly useful in categorical data or to identify the most likely value in a dataset.

**Q3. Measure the three measures of central tendency for the given height data: [178,177,176,177,178.2,178,175,179,180,175,178.9,176.2,177,172.5,178,176.5]**

**SOLUTION:**

**1. Mean (Average):**

To calculate the mean, sum up all the values and divide by the number of values.

Mean=∑Values/Number of Values

Step 1: Add up all the values:

178+177+176+177+178.2+178+175+179+180+175+178.9+176.2+177+172.5+178+176.5=2832.3

Step 2: Count the number of values in the dataset:

Number of Values=16

Step 3: Divide the sum by the number of values:

Mean=2832.3/16
= 177.01875

So, the mean is approximately 177.02 cm.

**2. Median:**

The median is the middle value in an ordered dataset. If there is an even number of values, it’s the average of the two middle values.

Step 1: Order the data from smallest to largest:

[172.5,175,175,176,176.2,176.5,177,177,177,177,178,178,178,178.2,178.9,179,180]

Step 2: Find the middle position. Since there are 16 values (an even number),

the median is the average of the 8th and 9th values.

The 8th value is 177

The 9th value is 177

Step 3: Calculate the median:

Median = 177 + 177 /2=177

So, the median is 177 cm.

**3. Mode:**

The mode is the value that appears most frequently in the dataset.

Looking at the ordered data:

[172.5,175,175,176,176.2,176.5,177,177,177,177,178,178,178,178.2,178.9,179,180]

The value 177 appears 3 times, which is more frequent than any other value.

So, the mode is 177 cm.

In [4]:
import numpy as np
from scipy import stats
data=[178,177,176,177,178.2,178,175,179,180,175,178.9,176.2,177,172.5,178,176.5]
mean=np.mean(data)
print('mean=',mean)
median=np.median(data)
print('median=',median)
mode=stats.mode(data)
print('mode=',mode)

mean= 177.01875
median= 177.0
mode= ModeResult(mode=177.0, count=3)


**Q4. Find the standard deviation for the given data: [178,177,176,177,178.2,178,175,179,180,175,178.9,176.2,177,172.5,178,176.5]**

**SOLUTION:**

[178,177,176,177,178.2,178,175,179,180,175,178.9,176.2,177,172.5,178,176.5]

Step 1: Calculate the Mean (Average)

The mean is calculated by adding up all the values and dividing by the number of values.

Mean =∑ Values /Number of Values

=2832.3/16

=177.02

Step 2: Calculate the Squared Differences from the Mean
For each value, subtract the mean and square the result.

(178−177.02)**2=0.9604

(177−177.02)**2=0.0004

(176−177.02)**2=1.0404

(177−177.02)**2=0.0004

(178.2−177.02)**2=1.3924

(178−177.02)**2=0.9604

(175−177.02)**2=4.0804

(179−177.02)**2=3.9204

(180−177.02)**2=8.9604

(175−177.02)**2=4.0804

(178.9−177.02)**2=3.5284

(176.2−177.02)**2=0.6724

(177−177.02)**2=0.0004

(172.5−177.02)**2=20.2504

(178−177.02)**2=0.9604

(176.5−177.02)**2=0.2704

Step 3: Sum of the Squared Differences
Now, sum all the squared differences:

0.9604+0.0004+1.0404+0.0004+1.3924+0.9604+4.0804+3.9204+8.9604+4.0804+3.5284+0.6724+0.0004+20.2504+0.9604+0.2704=50.0778


Step 4: Calculate the Variance

The variance is the average of the squared differences. For a sample, we divide by n−1 (where 𝑛 is the number of data points).

Variance =50.0778/16− 1
= 50.0778/15
= 3.3385

Step 5: Calculate the Standard Deviation
The standard deviation is the square root of the variance:

Standard Deviation
= ROOT(3.3385)
≈ 1.788

Final Answer:
The standard deviation for the given data is approximately 1.80 cm.
  


In [1]:
import numpy as np
data=[178,177,176,177,178.2,178,175,179,180,175,178.9,176.2,177,172.5,178,176.5]
np.std(data)

1.7885814036548633

**Q5. How are measures of dispersion such as range, variance, and standard deviation used to describe the spread of a dataset? Provide an example.**

**SOLUTION:**

**1. Range**

The range is the simplest measure of dispersion. It is the difference between the largest and smallest values in a dataset.

Formula:

Range=Maximum Value−Minimum Value

Example: Suppose we have the following heights (in cm) of 5 people:

165,170,172,180,175

Maximum value = 180

Minimum value = 165

Range=180−165=15

Interpretation: The range tells us that the height varies by 15 cm between the shortest and tallest person.

**2. Variance**
The variance measures the average of the squared differences between each data point and the mean. It provides insight into how far each data point in the dataset is from the mean.

Formula: For a sample of n values:

Variance= (∑(Xi​− m)**2)/n−1

where

Xi is each data point, and m is the mean.

Example: Consider the same height data:

165,170,172,180,175

Mean height =  165+170+172+180+175

 =172.4

Sum of squared differences = 54.76 + 5.76 + 0.16 + 57.76 + 6.76 = 125.2
Variance = 125.2/5−1

=125.2/4
= 31.3

Interpretation: The variance of 31.3 shows that the heights deviate from the mean by an average of approximately 31.3 squared units (cm²). However, because the units are squared, it’s harder to interpret directly.

Range gives a quick sense of the spread between the maximum and minimum values but doesn’t account for the distribution of values in between.


Variance and standard deviation consider how all values in the dataset deviate from the mean, providing a more comprehensive understanding of the data spread. The standard deviation is easier to interpret because it's in the same units as the data, while variance is in squared units.

**Q6. What is a Venn diagram?**

**SOLUTION:**

A Venn diagram is a visual tool used to show the relationships between different sets, typically represented by overlapping circles. It illustrates how sets intersect, differ, and relate to each other, helping to represent concepts like union, intersection, difference, and complement in a simple, visual way.

**Q7. For the two given sets A = (2,3,4,5,6,7) & B = (0,2,6,8,10). Find: (i) A B (ii) A ⋃ B**

**SOLUTION:**

A∩B={2,6}

A∪B={0,2,3,4,5,6,7,8,10}

**Q8. What do you understand about skewness in data?**

**SOLUTION:**

Skewness in data refers to the asymmetry or lack of symmetry in the distribution of data values around its mean. It describes how the data values are distributed in relation to the mean and whether they are concentrated more on one side of the mean than the other.

**Types of Skewness:**

**Symmetrical Distribution (No Skewness):**

In a perfectly symmetrical distribution, the left and right sides of the distribution are mirror images.

The mean, median, and mode are all the same or very close to each other.

Example:

A normal distribution (bell curve) is perfectly symmetrical.

**Positive Skewness (Right-Skewed):**

In a positively skewed distribution, the tail on the right-hand side is longer or fatter than the left side.

Most of the data points are concentrated on the left, with a few larger values pulling the mean to the right.

In this case: Mean > Median > Mode.

Example:

Income distribution, where a few individuals may have significantly higher incomes, pulling the average up.

**Negative Skewness (Left-Skewed):**

In a negatively skewed distribution, the tail on the left-hand side is longer or fatter than the right side.

Most of the data points are concentrated on the right, with a few smaller values pulling the mean to the left.

In this case: Mean < Median < Mode.

Example:

 Age of retirement, where most people retire around a specific age, but some may retire much earlier, pulling the mean to the left.

**How to Measure Skewness:**

Skewness coefficient: Skewness is quantified using a skewness coefficient, with the following interpretations:

If skewness > 0, the distribution is positively skewed.

If skewness < 0, the distribution is negatively skewed.

If skewness = 0, the distribution is symmetrical (no skew)


**Q9. If a data is right skewed then what will be the position of median with respect to mean?**

**SOLUTION:**

If a dataset is right-skewed (positively skewed), the median will be less than the mean.

 Mean > Median > Mode.

**Q10. Explain the difference between covariance and correlation. How are these measures used in statistical analysis?**

**SOLUTION:**

# **1. Definition:**

**Covariance:**

Covariance measures how two variables change together. It indicates the direction of the linear relationship between the variables.

If both variables increase together, the covariance is positive. If one variable increases while the other decreases, the covariance is negative.

**Correlation:**

Correlation quantifies the strength and direction of the linear relationship between two variables, standardizing the measure of covariance.

Correlation values are confined to a range between -1 and 1, where -1 indicates a perfect negative correlation, 1 indicates a perfect positive correlation, and 0 indicates no correlation.

# **2. Scale:**

**Covariance:**

The value of covariance can range from negative infinity to positive infinity. The magnitude is affected by the units of the variables, which makes interpretation difficult.

**Correlation:**

Correlation values range from -1 to 1, making them easier to interpret. A value of 0 indicates no linear relationship, while values closer to -1 or 1 indicate stronger relationships.

# **3. Interpretation:**

**Covariance:**

A positive covariance indicates that the variables tend to increase or decrease together, while a negative covariance indicates that as one variable increases, the other tends to decrease. However, it does not provide information about the strength of the relationship.

**Correlation:**

Correlation provides both the direction (positive or negative) and strength of the relationship. It indicates how closely the two variables follow a linear relationship.

# **Usage in Statistical Analysis:**

**Covariance:**

Used in portfolio theory in finance to measure how different assets move together.

Helps to identify relationships in data but is less frequently used in reporting results due to its interpretation difficulties.

**Correlation:**

Widely used in various fields such as economics, psychology, biology, and social sciences to understand relationships between variables.

Used to assess the strength of relationships in regression analysis, hypothesis testing, and predictive modeling.

Correlation coefficients are often reported to convey how strongly two variables are related.


while both covariance and correlation provide insights into the relationship between two variables, correlation is more commonly used due to its standardized scale and easier interpretation. Covariance gives a sense of direction but lacks clarity in terms of strength and comparative analysis.

**Q11. What is the formula for calculating the sample mean? Provide an example calculation for a dataset.**

**SOLUTION:**

The sample mean is a measure of central tendency that provides an average value for a dataset. The formula for calculating the sample mean (M) is:

M =∑Xi/n

Where:
M = Sample mean

Xi= Each individual data point in the dataset

n = Total number of data points in the sample

∑ = Summation notation, which means to add up all the individual data points

Example Calculation:

Let’s calculate the sample mean for the following dataset:

Dataset: {4,8,6,5,3}

Sum the Data Points:

∑ Xi=4+8+6+5+3=26

Count the Number of Data Points:

n=5     

(since there are 5 data points)

mean M =26/5=5.2


**Q12. For a normal distribution data what is the relationship between its measure of central tendency?**

**SOLUTION:**

In a normal distribution, the three measures of central tendency —mean, median, and mode
are equal and located at the center of the distribution.

Mean=Median=Mode

**Q13. How is covariance different from correlation?**

**SOLUTION:**

# **1. Definition:**

**Covariance:**

Covariance measures how two variables change together. It indicates the direction of the linear relationship between the variables.

If both variables increase together, the covariance is positive. If one variable increases while the other decreases, the covariance is negative.

**Correlation:**

Correlation quantifies the strength and direction of the linear relationship between two variables, standardizing the measure of covariance.

Correlation values are confined to a range between -1 and 1, where -1 indicates a perfect negative correlation, 1 indicates a perfect positive correlation, and 0 indicates no correlation.

# **2. Scale:**

**Covariance:**

The value of covariance can range from negative infinity to positive infinity. The magnitude is affected by the units of the variables, which makes interpretation difficult.

**Correlation:**

Correlation values range from -1 to 1, making them easier to interpret. A value of 0 indicates no linear relationship, while values closer to -1 or 1 indicate stronger relationships.

# **3. Interpretation:**

**Covariance:**

A positive covariance indicates that the variables tend to increase or decrease together, while a negative covariance indicates that as one variable increases, the other tends to decrease. However, it does not provide information about the strength of the relationship.

**Correlation:**

Correlation provides both the direction (positive or negative) and strength of the relationship. It indicates how closely the two variables follow a linear relationship.

**Q14. How do outliers affect measures of central tendency and dispersion? Provide an example.**

# **1. Effect on Measures of Central Tendency:**

**Mean:**

The mean is highly sensitive to outliers because it takes every data point into account when calculating the average. Even a single extreme outlier can drastically change the mean.

Example: Consider the data set

{10,12,11,13,15,90}.

The outlier here is 90.

Without the outlier: Mean = 10+12+11+13+15/5=12.2

With the outlier: Mean = 10+12+11+13+15+90/6 =25.17

The outlier significantly inflates the mean.

**Median:**

The median is less sensitive to outliers because it depends only on the middle value(s) of the ordered data, not the magnitude of any individual data point.
Example: For the same data set

{10,12,11,13,15,90}:

Without the outlier: Median = 12

With the outlier: Median = 12.5

The median changes very little, making it a more robust measure when outliers are present.

**Mode:**

The mode is not affected by outliers because it is simply the most frequent value in the dataset. Outliers are unlikely to affect the most common data point unless the outlier occurs more frequently than other values.
Example: In the dataset

{10,12,11,13,15,90},

the mode does not exist (no repeated values), but if we had a mode (say in a different dataset), an outlier would not alter it.

# **2. Effect on Measures of Dispersion:**

**Range:**

The range is the difference between the largest and smallest values. Since the range depends only on the extreme values, an outlier can dramatically increase the range.

Example: For {10,12,11,13,15,90}:

Without the outlier: Range =  15−10=5

With the outlier: Range = 90−10=80

The outlier drastically increases the range.


**Variance and Standard Deviation:**

Both variance and standard deviation measure how spread out the data is around the mean. Since these metrics use the squared deviations from the mean, outliers can disproportionately increase the values, making the dataset appear more dispersed than it actually is.

Example: In the dataset

{10,12,11,13,15,90}:

Without the outlier, the variance and standard deviation would be relatively small.

With the outlier, the squared deviations of 90 from the mean will be very large, inflating both the variance and standard deviation.

Specifically, for the above data with the outlier, the standard deviation will increase significantly:

Without outlier: SD = 1.79

With outlier: SD = 29.57